dataland-internal-storage/src/test/kotlin/db/migration/V10__MigrateRefactoredDataPointClassesLksgTest.kt

```diff
@@ -6,12 +6,15 @@ import org.junit.jupiter.api.Test
 class V10__MigrateRefactoredDataPointClassesLksgTest {
     @Test
     fun `test migration of refactored data point classes in Lksg framework`() {
+        val dataType = "lksg"
         TestUtils().testMigrationOfSingleDataset(
+            dataType,
             "V10/V10_OriginalMockDatabaseEntry.json",
             "V10/V10_expectedTransformedMockDatabaseEntry.json",
             V10__MigrateRefactoredDataPointClassesLksg()::migrateRefactoredDataPointClasses,
         )
         TestUtils().testMigrationOfSingleDataset(
+            dataType,
```

Comment:
why do you touch existing migration scripts? they won't be executed anyways, so please write a new one